src/blockchain/config.ts

```diff
@@ -44,8 +44,8 @@ export const tokens = asMap('symbol', [
     maxSell: '10000000',
     name: 'Wrapped Ether',
     icon: SvgImageSimple(ethSvg),
-    iconInverse: SvgImageSimple(ethInverseSvg),
-    iconCircle: SvgImageSimple(ethCircleSvg),
+    iconInverse: SvgImageSimple(ethCircleSvg),
```

Comment:
I think this change impacts icons on notifications (see vis reg for storybook)

Reply:
Actually it's not broken. WETH should have this icon. It's the inversed icon of ETH. It's all good.

![image](https://user-images.githubusercontent.com/23715025/55387342-8e429f00-5531-11e9-9f1e-58b23135e383.png)

![image](https://user-images.githubusercontent.com/23715025/55387365-9f8bab80-5531-11e9-843e-9c9bc3400c2b.png)

Reply:
I was refering to this: https://artifacts.codechecks.io/166230898/f42cfb87b3de11d853d299b0c926404a4f958d9a/storybook-vis-reg-report/index.html